src/app/helptext/system/reporting.ts

```diff
@@ -0,0 +1,20 @@
+import { T } from "app/translate-marker";
+
+export const helptext = {
+cpu_in_percentage_placeholder: T('Report CPU usage in percentage'),
+cpu_in_percentage_tooltip: T('Set to display CPU usage as percentages in Reporting.'),
+
+graphite_placeholder: T('Remote Graphite Server Hostname'),
+graphite_tooltip: T('Enter the IP address or hostname of a remote server\
+ running Graphite.'),
+    
+graph_age_placeholder: T('Graph Age'),
+graph_age_tooltip: T(''),
+
+graph_points_placeholder: T('Graph Points'),
+graph_points_tooltip: T(''),
+
+confirm_rrd_destroy_placeholder: T('Confirm RRD Destroy'),
+confirm_rrd_destroy_tooltip: T(''),
```

Comment:
Confirm RRD Destroy:

"Destroy the reporting database. Required for changes to <b>Graph Age</b> and <b>Graph Points</b> to take effect."